modules/Microsoft.VirtualMachineImages/imageTemplates/deploy.bicep

```
@@ -147,11 +164,13 @@ resource imageTemplate 'Microsoft.VirtualMachineImages/imageTemplates@2020-02-14
     vmProfile: {
       vmSize: vmSize
       osDiskSizeGB: osDiskSizeGB
+      userAssignedIdentities: !empty(vmUserAssignedIdentities) ? vmUserAssignedIdentities : []
```

The default value is already an empty array, so the condition is not needed, plus given the other comment on the parameter name above
```suggestion
      userAssignedIdentities: userAssignedIdentities
```